libs/openpgm/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2021 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openpgm
+PKG_VERSION:=5.3.128
+PKG_RELEASE:=1
+
+PKG_SOURCE:=release-5-3-128.tar.gz
+PKG_SOURCE_URL:=https://github.com/steve-o/openpgm/archive
+PKG_HASH:=8d707ef8dda45f4a7bc91016d7f2fed6a418637185d76c7ab30b306499c6d393
+
+PKG_MAINTAINER:=Ye Holmes <yeholmes@outlook.com>
+PKG_LICENSE:=LGPL-2.1-or-later
+PKG_LICENSE_FILES:=LICENSE
+PKG_CPE_ID:=cpe:/a:openpgm:openpgm
+
+PKG_FIXUP:=autoreconf
```

Comment:
PKG_INSTALL and PKG_BUILD_PARALLEL look good here. That way you can get rid of those nasty paths below.

Reply:
The post-move after tarball extraction is not good, however, by looking into include/package*.mk, I found another method to avoid the post-move, and handling the subdirectory nicely, by defining `MAKE_PATH` variable. I will post the patch shortly.